File: VB/Dennis.UserViewVariants/Module.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp
Imports System.Collections.Generic

Namespace Dennis.UserViewVariants
	Public NotInheritable Partial Class UserViewVariantsModule
		Inherits ModuleBase
		Public Sub New()
			ModelDifferenceResourceName = "Model.DesignedDiffs"
			InitializeComponent()
		End Sub
	End Class
End Namespace

File: CS/Dennis.UserViewVariants/Module.cs
using System;
using DevExpress.ExpressApp;
using System.Collections.Generic;

namespace Dennis.UserViewVariants {
    public sealed partial class UserViewVariantsModule : ModuleBase {
        public UserViewVariantsModule() {
            ModelDifferenceResourceName = "Dennis.UserViewVariants.Model.DesignedDiffs";
            InitializeComponent();
        }
    }
}
